Makefile

```diff
@@ -13,10 +13,19 @@ help: ## Display this help.
 	@$(UV) venv
 	@$(UV) pip install --requirement pyproject.toml
 
+LOCAL_PYSQUARED ?= ""
+
 .PHONY: download-libraries
 download-libraries: .venv ## Download the required libraries
 	@echo "Downloading libraries..."
-	@$(UV) pip install --requirement lib/requirements.txt --target lib --no-deps --upgrade --quiet
+	@$(UV) pip install --requirement lib/requirements.txt --target lib --no-deps --upgrade --quiet; \
+
+	@if [ -n "$(LOCAL_PYSQUARED)" ]; then \
+		$(UV) pip install $(LOCAL_PYSQUARED) --target lib --no-deps --upgrade --quiet; \
+	else \
+		$(UV) pip install git+https://github.com/proveskit/pysquared --target lib --no-deps --upgrade --quiet; \
```

Comment:
Hmm I would prefer to see this in `requirements.txt` at a minimum we should lock the version but maybe while we're in the midst of this transition to multiple repos, this is OK.